SQLiteStudio3/guiSQLiteStudio/translations/guiSQLiteStudio_zh_CN.ts

```diff
@@ -2167,7 +2169,8 @@ All objects from this group will be moved to parent group.</source>
     <message>
       <location filename="../dbtree/dbtree.cpp" line="1705"/>
       <source>Remainig objects from deleted group will be moved in place where the group used to be.</source>
-      <translation type="unfinished">Remainig objects from deleted group will be moved in place where the group used to be.</translation>
+      <translatorcomment>被删除分组中的剩余子对象会移动到原来的位置</translatorcomment>
+      <translation>Remainig objects from deleted group will be moved in place where the group used to be.</translation>
```

Comment:
The <translation> here is still untranslated, whereas the translated text seems to be placed in the <translatorcomment> field. I believe this is also by mistake?